doc/api-extensions.md

```diff
@@ -2340,3 +2340,8 @@ the event as that's usually already covered by the `location` field.
 Adds a new `volatile.uuid` configuration key to all storage volumes, snapshots and buckets.
 This information can be used by storage drivers as a separate identifier besides the name
 when working with volumes.
+
+## `stateful_migration_default`
```

Comment:
The commit message needs to be in the form:

```
api: Adds <extension> API extension.
```

E.g. https://github.com/canonical/lxd/commit/8ad0503c01162a7de99408376ade2d8ddf167612